src/hooks/useApplicationActions.ts

```diff
@@ -444,7 +445,15 @@ const useApplicationActions = (
           'Error sending proposal. Please try again or contact support.',
         )
       }
-
+      const response = await getStateWaitMsg(messageCID)
+      if (
+        typeof response.data === 'object' &&
+        response.data.ReturnDec.Code !== 0
+      ) {
+        throw new Error(
+          `Error sending transaction. Please try again or contact support. Error code: ${response.data.ReturnDec.Code}`,
+        )
+      }
```

Comment:
This in invalid for multisig threshold >1
Also, we need to do the same for approval calls.